Managers/LuaMan.cpp

```diff
@@ -819,6 +819,7 @@ int LuaMan::Initialize() {
             .property("RecoilOffset", &MOSRotating::GetRecoilOffset)
 			.property("TravelImpulse", &MOSRotating::GetTravelImpulse, &MOSRotating::SetTravelImpulse)
 			.property("GibWoundLimit", (int (MOSRotating:: *)() const) &MOSRotating::GetGibWoundLimit, &MOSRotating::SetGibWoundLimit)
+			.property("GibSound", &MOSRotating::GetGibSound, &MOSRotating::SetGibSound, detail::null_type(), adopt(_2))
```

Comment:
As I mentioned in direct messages, you should merge development into your branch and add `PROPERTYOWNERSHIPSAFETYFAKER([ObjectType], SoundContainer, Set[NameHere]);` entries, then use those (with no adopt policies) when you have the chance.